std_msgs/Header header
ros_objslampp_ycb_video/ObjectClass[] classes
